recipes/json-schema-validator/all/conanfile.py

```diff
@@ -94,20 +94,19 @@ def package(self):
         self.copy("LICENSE", dst="licenses", src=self._source_subfolder)
         cmake = self._configure_cmake()
         cmake.install()
-        if tools.Version(self.version) < "2.1.0":
+        if scm.Version(self.version) < "2.1.0":
             self.copy("json-schema.hpp",
                       dst=os.path.join("include", "nlohmann"),
                       src=os.path.join(self._source_subfolder, "src"))
-        tools.rmdir(os.path.join(self.package_folder, "lib", "cmake"))
+        files.rmdir(self, os.path.join(self.package_folder, "lib", "cmake"))
 
         # TODO: to remove in conan v2 once cmake_find_package* generators removed
         self._create_cmake_module_alias_targets(
             os.path.join(self.package_folder, self._module_file_rel_path),
             {"nlohmann_json_schema_validator": "nlohmann_json_schema_validator::nlohmann_json_schema_validator"}
         )
 
-    @staticmethod
-    def _create_cmake_module_alias_targets(module_file, targets):
+    def _create_cmake_module_alias_targets(self, module_file, targets):
```

Comment:
```suggestion
    def _create_cmake_module_alias_targets(conanfile, module_file, targets):
```
I would prefer `conanfile` or anything else which is not `self`. As it's a static method, you are shadowing `self` using it as parameter name.

Reply:
Notice I removed the `@staticmethod` annotation...

Reply:
Sorry, I misread the diff.